### DISCUSSION TITLE
[3.7.0] Change Audio Source Not always Working

### DISCUSSION THREAD

**Discussion by Tomhero200** [KRTirtho/spotube#1574] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I cannot change the Audio Source in 3.7.0.
If I press on the change Source Button, it doesn't Show me any recommendations except the wrong one that's playing. If I then press the search Button and choose the correct video/Audio, it doesn't work.

### Expected Behavior

It should use the Audio selected by me.

### Steps to reproduce

1. Open the App on Version 3.7.0 2. Select a Song (In my case, it's "Places" by Martin Solveig) 3. Try to change the source

### Operating System

Android

### Spotube version

3.7.0

### Installation source

Play Store (Android)

### Additional information

It's working fine on 3.6.0.
Most Songs work fine with the default source.

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
It was probabaly a cached result. Click on the search icon. It'll show more results
![image](https://github.com/KRTirtho/spotube/assets/61944859/be617614-caf2-4c62-bfa5-1e2a897726f4)
![image](https://github.com/KRTirtho/spotube/assets/61944859/3251dedd-4697-4790-90af-1b322b5c075b)
